states/menu.py

```diff
@@ -5,22 +5,29 @@
 
 
 class MenuOptions(StateOptions):
+    """Variables for Menu options"""
```

Comment:
These might be variables, but that's kind of obvious. Specifically, look at how `MenuOptions(StateOptions)` tells you how this class extends off of `StateOptions`. If you look at `StateOptions`, you can see that it is an `enum`. Therefore, `MenuOptions` is also an enum, and probably symbolizes the different options a player can choose in the menu.

Reply:
Changed to `Enum values for options available in the menu`